source/client/gui/components/ScrolledSelectionList.cpp

```diff
@@ -276,3 +275,10 @@ void ScrolledSelectionList::setRenderHeader(bool b, int i)
 		i = 0;
 	field_48 = i;
 }
+
+void ScrolledSelectionList::handleScroll(bool down)
+{
+	float diff = 5.0f * (down ? -1.0f : 1.0f);
```

Comment:
Feels weird.  Like, when you scroll up, and then down, it seems to "continue" scrolling in the direction I was first scrolling in. Maybe it should just instantly set the velocity.

Also I think you should add a timeout of a few ms where the world select is just loose (as if you were holding a finger on the scroll area)

Reply:
This comment is for RolledSelectionList? Problem doesn't appear with ScrolledSelectionList.

Reply:
Yes, oops... Comment is related to RolledSelectionList.